exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/traces/SpanMarshaler.java

```diff
@@ -37,6 +37,7 @@ final class SpanMarshaler extends MarshalerWithSize {
   private final SpanLinkMarshaler[] spanLinkMarshalers;
   private final int droppedLinksCount;
   private final SpanStatusMarshaler spanStatusMarshaler;
+  private final int flags;
```

Comment:
For completeness can you update the SpanLinkMarshaler as well?

Reply:
See original PR where this field was added for details: https://github.com/open-telemetry/opentelemetry-proto/pull/503/files